### DISCUSSION TITLE
让深度操作系统更易于访问的建议

### DISCUSSION THREAD

**Discussion by nepvibes** [linuxdeepin/developer-center#5720] [Developer Discussion | 开发者讨论]
edited this text on english  bellow [https://github.com/orgs/linuxdeepin/discussions/5720#discussioncomment-7066990](https://github.com/orgs/linuxdeepin/discussions/5720#discussioncomment-7066990)

**Comment by BLumia** [comment]
Hi, I don't understand the issue that you're trying to feedback, you can use English if preferred. Thanks :)

**Reply by nepvibes** [reply]
@BLumia please create a test repository for the DDE-dock custom plugin. Previous deepin plugins did not respect width. Test https://github.com/SeptemberHX/dde-shelloutput-plugin for reference.
